.github/workflows/cloud_build_failure_reporter.yml

```diff
@@ -134,16 +140,16 @@ jobs:
                           check.conclusion === 'success'
                       ) {
                           updateIssues(
-                              `[Passing periodic](${check.html_url}) at ${commit.html_url}.`
+                              `[Tests are passing](${check.html_url}) for commit [${commit.sha}](${commit.html_url}).`
                           );
                       } else if (check.status === 'in_progress') {
                           console.log(
                               `Check is pending ${check.html_url} for ${commit.html_url}. Retry again later.`
                           );
                       } else {
                           createOrCommentIssue(
-                              `[Cloud Build Failure Reporter] test failing for **${check.name}**`,
-                              `Cloud Build Failure Reporter found test failure for **${check.name}** ([link to check](${check.html_url})) at [link to commit](${commit.html_url}). Please fix the error and then close the issue after the **${check.name}** test passes.`
+                              `Cloud Build Failure Reporter: ${check.name} failed`,
+                              `Cloud Build Failure Reporter found test failure for **${check.name}** ([check.name](${check.html_url})) at [commit sha](${commit.html_url}). Please fix the error and then close the issue after the **${check.name}** test passes.`
```

Comment:
```suggestion
                              `Cloud Build Failure Reporter found test failure for [**${check.name}** ](${check.html_url}) at [${commit.sha}](${commit.html_url}). Please fix the error and then close the issue after the **${check.name}** test passes.`
```

We want to provide more info like the commit sha and make sure it's nicely formatted for the user.